### ISSUE TITLE
Please push updated 6.x babel-traverse release to support flow type spread eslint

### ISSUE THREAD

**Issue by rosskevin** [babel/babel#5812]
#5653 Port flow object spread from #418 to 6.x 

^This PR backports a fix to 6.x, and it was released, but it appears this has had no affect on `babel-eslint` and related plugins because they are using an older `babel-traverse` without the code change (I'm no expert so please don't take this as such).

From babel-eslint issue: 
https://github.com/babel/babel-eslint/issues/473#issuecomment-305655373

> @rosskevin I looked into this.
>
> The problem is that there is no range information. This is because babel-eslint is not traversing into the ObjectTypeSpreadProperty node arguments. This node type was added to babel-types by this PR babel/babel#5653 but there has been no release and this means that the package babel-traverse is not including the newest type definitions.
>
> To fix this I think there just has to be a new release of babel v6.

Here's a repo where I've attempted to use 7.x codeline to get eslint to work again, with no luck:
https://github.com/rosskevin/babel-eslint-473-flow-spread

Using 7.x may be too much to ask at this point, as you can see from the linked `babel-eslint` issue, I use quite a few other plugins which are relying on 6.x `babel-traverse`.  I'm happy to pursue those plugins and get them patched up to date on the 6.x line, but getting several to get a 7.x release out immediately may be asking too much.

## TL;DR
I'm just trying to get eslint to work with flow spread.  I need them both, and am painted into a corner.  Any advice, help, or workarounds are appreciated.

**Comment by hzoo** [comment]
releasing a new version now.

We can just update babel-eslint to 7.x

**Comment by rosskevin** [comment]
I'm having problems using flow object type spread (which I now depend on greatly) because so many transitive dependencies still depend on 6.x, which doesn't have everything necessary to support it released.  I'm pushing and PRing things for 7.x support, but quickly getting frustrated as I chase 6.x dependencies without the object type spread support.

**Comment by rosskevin** [comment]
Thank you!

https://github.com/babel/babel/releases/tag/v6.25.0
